vms/platformvm/config/internal.go

```diff
@@ -37,7 +36,7 @@ type Internal struct {
 
 	// Static fees are active before Etna
 	CreateAssetTxFee uint64 // Override for CreateSubnet and CreateChain before AP3
-	StaticFeeConfig  txfee.StaticConfig
+	TxFee            uint64
```

Comment:
Can we get rid of these entries as well? I think any time we use them we could just replace the value with `0`.